SubterfugeCoreTest/MapGenerator.test.cs

```diff
@@ -339,7 +340,29 @@ public void MinimumOutpostDistanceRespected()
             Outpost outpost2 = generatedOutposts[1];
 
             float distance = (outpost1.GetCurrentPosition()-outpost2.GetCurrentPosition()).Magnitude();
-            Assert.IsTrue(distance > config.MinimumOutpostDistance);
+            Assert.IsTrue(distance >= config.MinimumOutpostDistance);
+        }
+
+        [TestMethod]
+        public void AllOutpostsHaveUniqueNames()
+        {
+            List<Player> players = new List<Player>();
+            players.Add(new Player(1));
+            players.Add(new Player(2));
+            players.Add(new Player(3));
+            
+            GameConfiguration config = new GameConfiguration(players);
+            Assert.IsNotNull(config);
+            config.Seed = 1234;
```

Comment:
I think the seed should be randomised to have a proper test. It might make the test flakey (in the sense that it _could_ fail sometimes and pass other times) but it'll be a more resilient test.

I also think there should be a test of this kind where the total list of outpost names is exhausted and the randomiser has to be used -- that way you test that functionality too. With this test, if I read it correctly, there will be only 9 outposts in total, which (of course) all can be assigned a name from the current name list.

Reply:
Resolved this by randomly determining the seed